### ISSUE TITLE
[Bug] Ghost Targeting Indicators

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#2353]
### Short Description of the issue:
I built a drydock style shipyard to test out the new shipyard block, and tried to dock with it. As you move around the drydock, "ghost" targeting indicators pop up and follow the ship around the drydock. Video attached.
This might be caused by the fact that the shipyard is on a ship, not a station.

As a side note, is this the kind of drydock shipyard you'd like to see? It's still WIP and needs a docking connector, but I can complete it with a little more time

### Steps to reproduce the issue:
Spawn in the attached Drydock blueprint and try to dock with it.

### Environment
- Starship EVO version: 20w23d
- Graphics Card: RX 580

### Additional information:

[Starship EVO Ghost Target Indicator Bug.zip](https://github.com/tsunamayo/Starship-EVO/files/4742248/Starship.EVO.Ghost.Target.Indicator.Bug.zip)
[Drydock.zip](https://github.com/tsunamayo/Starship-EVO/files/4742251/Drydock.zip)

**Comment by tsunamayo** [comment]
yes that correct, except I think if it could fit a cubic ship of a bigger size that would be great! I am looking at the bug atm.

**Comment by tsunamayo** [comment]
also output.log could help! Thanks

**Comment by tsunamayo** [comment]
@ProPeach I dont see those ghost - did you unspawn the shipyard or something? Thanks

**Comment by Majikmonster** [comment]
I'm getting this bug as well. It seems to happen when a shipyard unit is built and then deleted, when a station part with a shipyard is relocated with the connector tool, and when a module with a shipyard is unspawned. The indicators also follow the player's ship after all of the entities with shipyards have been despawned. Restarting the game makes them disappear.

It might be hard to see in the image, but there are multiple indicators of varying size and distance in front of the ship. None of those were in that direction until I turned around, and they followed me when I backed up the ship.
![20200608084739_1](https://user-images.githubusercontent.com/51390907/84125066-05af7080-aa0a-11ea-9be1-a9395d7c4a58.jpg)
